eap-xp/image.yaml

```diff
@@ -50,8 +50,8 @@ modules:
             version: "11"
           - name: jboss.container.maven.35.bash
             version: "3.5"
-          - name: eap-xp-10-env
-            version: "1.0.3"
+          - name: eap-xp-20-env
+            version: "2.0.0"
```

Comment:
Just a note, as this module is specific to EAP XP2 (`eap-xp-20-env`), its version should not be tied to 2.0 (it can start with `1.0.0`).

Reply:
Do we want to start with :1.0? I can update these at the same time as making the other changes below.

Reply:
ok, let's then start at "1.0". thanks!